lib/ruby_memoized/memoizer.rb

```diff
@@ -7,13 +7,13 @@ def initialize(context, method)
       @method = method
     end
 
-    def call(*args, &block)
-      return cache[[args, block]] if cache.has_key?([args, block])
-      cache[[args, block]] = context.send(method, *args, &block)
+    def call(*args, **kwargs, &block)
```

Comment:
I think this works for updating the cache, but does this defining of the methods that you do [here](https://github.com/kklimuk/ruby_memoized/blob/master/lib/ruby_memoized.rb#L37) also need to receive the same type of update to cover the kwargs case?

Reply:
🤦 great point! fixed